src/app/navbar/navbar.component.html

```diff
@@ -1,9 +1,9 @@
 <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
-    <nav class="navbar navbar-fixed-top navbar-default">
-        <div class="container-fluid">
+    <nav class="navbar navbar-fixed-top navbar-default navbar-custom">
+        <div class="container-fluid search_n_head">
```

Comment:
Use meaningful class name. Don't give it as `search_n_head`. Also avoid writing in underscores. You can give it as `search-head`